_commands/basics/su.md

```diff
@@ -0,0 +1,70 @@
+---
+---
+
+su
+---
+
+`su` (substitute user) is a command to change a login session's owner
+without the owner having to first log out of the session. (as specified 
+in the [su Command](https://www.techonthenet.com/linux/commands/su.php)). In
+other words, it is called the switch user command
+
+__Note:__ It is most commonly employed to change the ownership from
+an ordinary user to the administrative user. Therefore it is oftern
+refered as the superuser command.
+
+
+
+~~~ bash
+$ su root
+~~~
+
+<!--more-->
+
+### Example
+
+~~~ bash
+$ su root
+Password:
+$ sudo alex
```

Comment:
`sudo alex` would run the command `alex` as root. Is that what you are trying to do here?